src/commands/txCommands/util/index.js

```diff
@@ -325,6 +325,35 @@ const resources$ =
 		resourceInfo.sort(lastUpdateComparator).map(resource => resource['slug'])
 	);
 
+const resourceStats$ = Rx.Observable.bindNodeCallback(
+	tx.statisticsMethods.bind(tx)
+);
+
+const resourceCompletion$ = resources$
+	.flatMap(Rx.Observable.from)
+	// get resource completion percentage
+	.flatMap(resource => resourceStats$(PROJECT, resource)
+		// reduce the amount of data we're working with
+		.map(resourceStat => Object.keys(resourceStat)
+			// filter out secondary locale tags. they don't matter for completion
+			.filter(key => packageConfig.secondaryLocaleTags.indexOf(key) === -1)
+			// reduce to object that only has incomplete languages
+			.reduce((localeCompletion, locale_tag) => {
+				resourceStat[locale_tag].completed !== '100%'
+					&& (localeCompletion[locale_tag] = resourceStat[locale_tag].completed);
+				return localeCompletion;
+			}, {})
+		)
+		.map(localeCompletion => [resource, localeCompletion])
+	);
+
+const resourcesIncomplete$ = resourceCompletion$
+	.filter(item => Object.keys(item[1]).length);
+
+const resourcesComplete$ = resourceCompletion$
+	.filter(item => Object.keys(item[1]).length === 0)
+	.map(([resource, lang_completion]) => resource);
```

Comment:
nitpick, but `lang_completion`  is called `localeCompletion` on line 347 - probably good to keep naming consistent just for greppability